### PR TITLE
Revise personality page

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
- Remove redundancy of component
- Manage each slider's value by array

### CODE REVIEW

File: src/components/Personality.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            minimumValue={0}
+            step={1}
+          />
+        </View>
+      )
+    );
+    return (
+      <ScrollView style={styles.scroll}>
+        <View style={styles.titleContainer}>
+          <Text style={styles.title}>Let’s figure out your
```
Comment (ochanje210) [inline src/components/Personality.js]: this new line in the middle of the text content is very annoying lol

Comment (ochanje210) [reply src/components/Personality.js]: but good work. Ill give u my LGTM

Comment (ifndefdeadmau5) [reply src/components/Personality.js]: :-) Oh Thanks. I did not notice it.

File: src/components/Personality.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            minimumValue={0}
+            step={1}
+          />
+        </View>
+      )
+    );
+    return (
+      <ScrollView style={styles.scroll}>
+        <View style={styles.titleContainer}>
+          <Text style={styles.title}>Let’s figure out your personality !</Text>
```
Comment (ethkim) [inline src/components/Personality.js:61]: I think you want to NOT leave a space between "personality" and "!" (unless the rendering makes it look weird).

**Review by ethkim — APPROVED** [review]
LGTM modulo the nit above. Thanks!!

File: src/components/Personality.js (hunk 2 above)
Comment (ochanje210) [reply src/components/Personality.js:61]: agreed

Comment (ifndefdeadmau5) [reply src/components/Personality.js:61]: image
This one is from our design guide, so... I added a space to follw the guide

Comment (ochanje210) [reply src/components/Personality.js:61]: ohhh so the space you added in the code actually makes that space in the design guide as well? I did not know it.

**Comment by ace0625** [comment]
lgtm

**Pull request merged**